livekit_models.proto

```diff
@@ -145,3 +145,25 @@ message ParticipantTracks {
   string participant_sid = 1;
   repeated string track_sids = 2;
 }
+
+// details about the client
+message ClientInfo {
+  enum SDK {
+     UNKNOWN = 0;
+     JS = 1;
+     IOS = 2;
```

Comment:
Should we change this to SWIFT?

Reply:
first step in iOS -> SWIFT conversion. I gotta get used to this!